packages/esm-patient-chart-app/src/clinical-views/components/encounter-tile/tile.scss

```diff
@@ -0,0 +1,96 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.title {
+  @include type.type-style('productive-heading-03');
+}
+
+.title:after {
+  content: '';
+  display: block;
+  width: layout.$spacing-07;
+  padding-top: layout.$spacing-01;
+  border-bottom: layout.$spacing-03 solid $brand-teal-01;
+}
+
+.tileTitle {
+  @include type.type-style('label-01');
+  display: block;
+  padding-top: layout.$spacing-05;
+}
+
+.columnContainer {
+  margin: 0px 0px layout.$spacing-06 layout.$spacing-06;
+  display: flex;
+  flex-direction: row;
+}
+
+.tileBoxColumn {
+  width: 100%;
+  display: 'flex';
+  flex-direction: 'row';
+}
+
+.tileSubTitle {
+  @include type.type-style('body-compact-01');
+}
+
+.tileValue {
+  @include type.type-style('body-long-01');
+  margin: layout.$spacing-05 layout.$spacing-10 layout.$spacing-06 0;
+}
+
+.tileBox {
+  width: 25%;
```

Comment:
This seems like a bad assumption here. Surely we can use flex-box to figure out the right width for this element?

Reply:
Otherwise, this widget winds up being much less flexible than it seems to be...